app/jobs/base.rb

```diff
@@ -69,6 +69,7 @@ def initialize(job_class:, opts:, db:, jid:)
 
           MethodProfiler.ensure_discourse_instrumentation!
           MethodProfiler.start
+          @data["live_slots_start"] = GC.stat[:heap_live_slots]
```

Comment:
I chose `live_slots_start` since this is what `mini_scheduler` uses 

https://github.com/discourse/mini_scheduler/blob/5aecec355779539f43fe66583aa2446fa51d0d64/lib/mini_scheduler/manager.rb#L132